planprogenerator/planproxml/signalxml.py

```diff
@@ -36,44 +38,44 @@ def get_signal_xml(signal):
              + f"              <Objektreferenzen/>" + "\n" \
              + f"              <Punkt_Objekt_Strecke>" + "\n" \
              + f"                <ID_Strecke>" + "\n" \
-             + f"                  <Wert>{signal.trip.trip_uuid}</Wert>" + "\n" \
+             + f"                  <Wert>{signal.trip.uuid}</Wert>" + "\n" \
              + f"                </ID_Strecke>" + "\n" \
              + f"                <Strecke_Km>" + "\n" \
              + f"                  <Wert>{trip_length}</Wert>" + "\n" \
              + f"                </Strecke_Km>" + "\n" \
              + f"              </Punkt_Objekt_Strecke>" + "\n" \
              + f"              <Punkt_Objekt_TOP_Kante>" + "\n" \
              + f"                <Abstand>" + "\n" \
-             + f"                  <Wert>{signal.distance_edge:.3f}</Wert>" + "\n" \
+             + f"                  <Wert>{signal.distance_previous_node:.3f}</Wert>" + "\n" \
```

Comment:
Just because I see it here. How is the `distance_previous_node` defined, especially in the case that the `Wirkrichtung` is `gegen`? The PlanPro documentation says `Abstand des Objekts vom Knoten A bezogen auf TOP_Kante in Metern`. In the case of a `gegen` signal I would expect that the previous node is node B of the top edge, then this line here would be wrong.

Reply:
Good point. I don't think it is well-defined, although “previous” already suggests that there is a direction applied, so we could define it a being the **distance to the previous node** in `Wirkrichtung` (A or B).

Reply:
We now decided to use the **PlanPro** definition.